pkg/apis/cluster/v1alpha1/machine_types.go

```diff
@@ -160,7 +160,7 @@ type MachineStatus struct {
 	LastOperation *LastOperation `json:"lastOperation,omitempty"`
 
 	// Phase represents the current phase of machine actuation.
-	// Eg. Pending, Running, Terminating, Failed etc.
+	// e.g. Pending, Running, Terminating, Failed etc.
```

Comment:
This is sentence and should be capitalized.

Reply:
Thanks @davidewatson - should be fixed :-)